features/peripheral_device_features.py

```diff
@@ -184,6 +184,44 @@ def _while_wait(self):
         import pygame
         pygame.event.clear()
 
+class EyeControl(object):
+    '''
+    this class implements a python cursor control task
```

Comment:
maybe we can delete this class at some point? can you make a comment that says it is only for testing